addons/survey/models/survey_survey.py

```diff
@@ -276,36 +278,30 @@ def _compute_has_conditional_questions(self):
         for survey in self:
             survey.has_conditional_questions = any(question.is_conditional for question in survey.question_and_page_ids)
 
-    @api.onchange('scoring_success_min')
-    def _onchange_scoring_success_min(self):
-        if self.scoring_success_min < 0 or self.scoring_success_min > 100:
-            self.scoring_success_min = 80.0
-
-    @api.onchange('scoring_type')
-    def _onchange_scoring_type(self):
-        if self.scoring_type == 'no_scoring':
-            self.certification = False
-            self.is_time_limited = False
+    @api.depends('scoring_type')
+    def _compute_certification(self):
+        for survey in self:
+            if survey.certification is None or survey.scoring_type == 'no_scoring':
+                survey.certification = False
 
-    @api.onchange('attempts_limit')
-    def _onchange_attempts_limit(self):
-        if self.attempts_limit <= 0:
-            self.attempts_limit = 1
+    @api.depends('access_mode', 'users_login_required')
+    def _compute_is_attempts_limited(self):
```

Comment:
There is already an _compute_is_attempts_limited defined ^^

Reply:
Oopsie.

It was actually introduced by a DBE commit on conditional questions, and this PR was created before that commit was merged.
But still, good catch sir!